### DISCUSSION TITLE
Ideas for changing Hazards or new ones

### DISCUSSION THREAD

**Discussion by ninadicara** [very-good-science/data-hazards#107] [Ideas]
This is a thread where people can post ideas for changes to the Hazards. 
This is not about changes to the documentation, but the actual Hazard itself and what it represents.

**Comment by ninadicara** [comment]
Thinking about tools being exclusionary and that not fitting anywhere in the current hazards. 
For example, data visualisation tools that are inaccessible to people who are blind or have color blindness.
These could go under 'difficult to understand' but I'm not sure that's the same theme - i.e. that it's excluding part of the community. 
Hmm need to think about it more

**Comment by NatalieZelenka** [comment]
This could fit under lacks community involvement?

**Reply by ninadicara** [reply]
I don't think it's the same thing because I see that more as being directly for researchers researching people who aren't involved in the research. So if it was research about sight loss and people with sight loss couldn't get involved because of exclusionary reasons I think that's different to someone producing a tool about the environmental costs of machine learning models (or something) that can't be used by someone with accessibility needs - does that make sense? 
This could also apply to tools that are paywalled too

**Comment by ninadicara** [comment]
Excellent discussion in the MozFest session about exploitative data creation practices. E.G. Using refugees to get data labelled - see article here https://restofworld.org/2021/refugees-machine-learning-big-tech/

None of the attendees felt it really fell into a current label and I agree that it aligns with lots of them but doesn't directly fit.
We had to cut off this discussion early but hoping to pick it back up in our next co-working call :)

**Reply by MelanieIStefan** [reply]
Related to that, I have been thinking of a label along the lines of "Lacks attribution" - if someone else's work (for instance, in the collection, labelling, entering, or analysis of data) has been used without their work being attributed or acknowledged. I feel like the labels around informed consent are more about whose data is being used, but less about whose labour is being used.

**Comment by MelanieIStefan** [comment]
I wonder if there should be a specific Data Hazard for "Includes data from experiments on non-human animals". There is an argument to be made that it falls under "Lacks Informed Consent", but I don't know that that is the best place for it. I think there is a distinction to make between situations where informed consent is in principle possible, but has not been obtained or even sought, and situations where informed consent is impossible. I think animal research falls into the latter category.  In addition, it might be useful for some downstream applications to have a specific flag for whether animal research is involved (even if we decide that in that case, the "no informed consent" should still be up). For instance, once the use of Data Hazard labels is relatively widespread, researchers on animal welfare or the three Rs could use the labels relatively easily to compare projects with respect to their use of animals over time or across fields. So, even if the "includes data from non-human animals" label would not be completely independent of the "lacks informed consent" label, it might be a useful thing to have.
